src/main/java/decodes/hdb/HDBRatingTable.java

```diff
@@ -73,16 +73,18 @@ public HDBRatingTable ()
 	public void findRating(Date valueDate)
 	throws DbCompException
 	{
-		String q = "select ratings.find_site_rating('" + ratingType + 
-		"'," + sdi + ", + to_date(?,'DD.MM.YYYY HH24:MI:SS')) from dual";
+		String q = "select ratings.find_site_rating(?,?," +
+		"to_date(?,'DD.MM.YYYY HH24:MI:SS')) from dual";
 		CallableStatement cstmt =  null;
 
 		try
 		{
 			cstmt = conn.prepareCall(q);
```

Comment:
if you move this to the try-with-resources syntax you can ditch the finally block bellow.
e.g. try( cstmt = conn.prepalCall(q); ) {